contrib/babelfishpg_common/sql/geography.sql

```diff
@@ -515,21 +539,54 @@ CREATE OR REPLACE FUNCTION sys.STDistance_helper(geog1 sys.GEOGRAPHY, geog2 sys.
 
 CREATE OR REPLACE FUNCTION sys.GEOGRAPHY_helper(bytea)
 	RETURNS sys.GEOGRAPHY
-	AS '$libdir/postgis-3','geography_from_binary'
+	AS '$libdir/postgis-3','LWGEOM_from_bytea'
 	LANGUAGE 'c' IMMUTABLE STRICT PARALLEL SAFE;
 
-CREATE OR REPLACE FUNCTION sys.bpcharToGeography_helper(sys.bpchar)
+CREATE OR REPLACE FUNCTION sys.bpcharToGeography_helper(sys.bpchar, integer)
 	RETURNS sys.GEOGRAPHY
-	AS '$libdir/postgis-3','geography_from_text'
+	AS '$libdir/postgis-3','LWGEOM_from_text'
 	LANGUAGE 'c' IMMUTABLE STRICT PARALLEL SAFE;
 
-CREATE OR REPLACE FUNCTION sys.varcharToGeography_helper(sys.varchar)
+CREATE OR REPLACE FUNCTION sys.varcharToGeography_helper(sys.varchar, integer)
 	RETURNS sys.GEOGRAPHY
-	AS '$libdir/postgis-3','geography_from_text'
+	AS '$libdir/postgis-3','LWGEOM_from_text'
 	LANGUAGE 'c' IMMUTABLE STRICT PARALLEL SAFE;
```

Comment:
Looks like this function is not being used anymore, if so then let's remove it.

Reply:
Done